### PR TITLE
Win installer

### PR DESCRIPTION
**Pull request by ismailsunni**
### What does it fix?
Update win installer to work on QGIS3.

Notes:
- Only install for the default profile
- The user will need to enable it manually

* Ticket: #
* Funded by: 
* Description: 
  * 

### Checklist:

- [ ] Commit messages are descriptive and explain the rationale for changes
- [ ] Add to the changelog in metadata.txt if it's a new feature
- [ ] Unit test for new code added
- [ ] Request someone to review or test your PR
### What does it fix?
Update win installer to work on QGIS3.

Notes:
- Only install for the default profile
- The user will need to enable it manually

* Ticket: #
* Funded by: 
* Description: 
  * 

### Checklist:

- [ ] Commit messages are descriptive and explain the rationale for changes
- [ ] Add to the changelog in metadata.txt if it's a new feature
- [ ] Unit test for new code added
- [ ] Request someone to review or test your PR

### CODE REVIEW

File: scripts/windows-install-builder.nsi [hunk 1]
```diff
@@ -24,7 +24,8 @@ Name "InaSAFE [[VERSION]]"
 OutFile "InaSAFE-[[VERSION]]-plugin.exe"
 
 # define installation directory
-InstallDir $DOCUMENTS\..\.qgis2\python\plugins\inasafe
+# TODO(IS): currently only install to default profile directory.
+InstallDir $DOCUMENTS\..\AppData\Roaming\QGIS\QGIS3\profiles\default\python\plugins\inasafe
```
Comment (Gustry) [inline scripts/windows-install-builder.nsi]: Users might not have the `default` profile folder (low probability). Is-it possible to check which folder exists in `AppData\Roaming\QGIS\QGIS3\profiles\` and ask in which folder to install?

File: scripts/windows-install-builder.nsi [hunk 2]
```diff
@@ -68,12 +69,12 @@ RequestExecutionLevel user
...
 
-LangString WELCOME_TITLE ${LANG_INDONESIAN} "Indonesian title"
+LangString WELCOME_TITLE ${LANG_INDONESIAN} "Instalatur InaSAFE [[VERSION]]"
 LangString WELCOME_TITLE ${LANG_ENGLISH} "InaSAFE [[VERSION]] Installer"
 !define MUI_WELCOMEPAGE_TITLE $(WELCOME_TITLE)
 
-LangString WELCOME_TEXT ${LANG_INDONESIAN} "FOO BAR"
-LangString WELCOME_TEXT ${LANG_ENGLISH} "This setup tool will guide you through the process of installing the InaSAFE plugin. Note that it does not install QGIS itself - you must do that separately."
+LangString WELCOME_TEXT ${LANG_INDONESIAN} "Perangkat ini akan memandu Anda "
+LangString WELCOME_TEXT ${LANG_ENGLISH} "Alat pengaturan ini akan memandu Anda melalui proses menginstal plugin InaSAFE. Perhatikan bahwa itu tidak menginstal QGIS sendiri - Anda harus melakukannya secara terpisah."
```
Comment (Gustry) [inline scripts/windows-install-builder.nsi]: This doesn't look like english readable ...

Comment (ismailsunni) [reply scripts/windows-install-builder.nsi]: Ah thanks @Gustry 
I missed it, trying to copy and paste, but got it wrong. I will update soon.

File: scripts/windows-install-builder.nsi (hunk 1 above)
Comment (ismailsunni) [reply scripts/windows-install-builder.nsi]: There must be. But it seems the time doesn't allow me to spend too long in this win installer.
I will assume that this installer is being used for beginner user (or intermediate), and most probably still have `default` profile and use it.

Non-beginner user will be able to install InaSAFE easily :)

**Pull request merged**